app/views/abstract_jobs/index.html.erb

```diff
@@ -61,7 +73,12 @@
         <% end %>
 
         <% if params[:status] == 'all' %>
-          <td><%= abstract_job.status.capitalize %></td>
+          <td>
+            <% status = abstract_job.status.capitalize %>
+            <span class="<%= status == 'Failed' ? 'failed' : '' %>">
```

Comment:
Extremely minor and no need to change, but maybe I'd call this `error` (more standard)

Reply:
updated to "warning" to match the color ;)